### PR TITLE
Add downtime rrule attribute.

### PR DESCRIPTION
**Pull request by phillip-dd**
Closes https://github.com/DataDog/terraform-provider-datadog/issues/393

### CODE REVIEW

File: datadog/resource_datadog_downtime_test.go [hunk 1]
```diff
@@ -279,6 +279,38 @@ func TestAccDatadogDowntime_WeekDayRecurring(t *testing.T) {
 	})
 }
 
+func TestAccDatadogDowntime_RRule(t *testing.T) {
+	accProviders, cleanup := testAccProviders(t, initRecorder(t))
```
Comment (therve) [inline datadog/resource_datadog_downtime_test.go:283]:
```suggestion
	accProviders, _, cleanup := testAccProviders(t, initRecorder(t))
```

Comment (phillip-dd) [reply datadog/resource_datadog_downtime_test.go]: Thanks, updated!

File: datadog/resource_datadog_downtime_test.go [hunk 2]
```diff
@@ -279,6 +279,38 @@ func TestAccDatadogDowntime_WeekDayRecurring(t *testing.T) {
 	})
 }
 
+func TestAccDatadogDowntime_RRule(t *testing.T) {
+	accProviders, _, cleanup := testAccProviders(t, initRecorder(t))
+	defer cleanup(t)
+	accProvider := testAccProvider(t, accProviders)
+
+	resource.Test(t, resource.TestCase{
```
Comment (therve) [inline datadog/resource_datadog_downtime_test.go:287]:
```suggestion
	resource.ParallelTest(t, resource.TestCase{
```

Comment (phillip-dd) [reply datadog/resource_datadog_downtime_test.go]: Done!

**Comment by Izzette** [comment]
@phillip-dd tysm for working on this, we'll try it out soon!

**Pull request merged**